lib/myfinance/resources/payable_account.rb

```diff
@@ -14,12 +14,19 @@ def pay(id, entity_id, params = {})
         end
       end
 
+      def undo_payment(id, entity_id)
+        http.put(endpoints(id, entity_id, :undo_payment)) do |response|
+          respond_with_object(response, 'payable_account')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.